config/routes.rb

```diff
@@ -41,9 +41,9 @@
     get 'rotate_key', to: 'ips#index'
   end
   resources :team_members, only: %i[index edit update destroy] do
+    resources :memberships, only: %i[edit update]
```

Comment:
Is this necessary in this PR? I was thinking that `:team_members` route can eventually just become `organisation/:id/memberships` but that could be in the next PR.